File: src/main/java/net/theevilreaper/vulpes/api/repository/BlockRepository.java
package net.theevilreaper.vulpes.api.repository;

import io.micronaut.data.mongodb.annotation.MongoRepository;
import io.micronaut.data.repository.PageableRepository;
import net.theevilreaper.vulpes.api.model.BlockModel;

@MongoRepository
public interface BlockRepository extends PageableRepository<BlockModel, String> {
}

File: src/main/kotlin/net/theevilreaper/vulpes/api/repository/BlocKRepository.kt
package net.theevilreaper.vulpes.api.repository

import net.theevilreaper.vulpes.api.model.BlockModel
import org.springframework.data.mongodb.repository.MongoRepository


/**
 * @author theEvilReaper
 * @version 1.0.0
 * @since
 **/

interface BlocKRepository : MongoRepository<BlockModel, String>